src/interventions/Interventions.jl

```diff
@@ -5,110 +5,100 @@ Base.@kwdef struct Intervention{N,P,P2} <: EcoModel
     # Bounds are defined as floats to maintain type stability
     guided::N = Param(
         0;
-        ptype="categorical",
+        ptype="integer",
```

Comment:
This was set to categorical for a reason in #491 ?

Reply:
Thanks for spotting I missed this when fixing conflicts during the rebase